schematic/store/synapse.py

```diff
@@ -1401,14 +1402,16 @@ def format_row_annotations(
             else:
                 if isinstance(anno_v, float) and np.isnan(anno_v):
                     annos[anno_k] = ""
-                elif (
-                    isinstance(anno_v, str)
-                    and re.fullmatch(csv_list_regex, anno_v)
-                    and rule_in_rule_list(
-                        "list", dmge.get_node_validation_rules(anno_k)
-                    )
-                ):
-                    annos[anno_k] = anno_v.split(",")
+                elif isinstance(anno_v, str) and re.fullmatch(csv_list_regex, anno_v):
+                    rule_list = []
+                    if annotation_keys == "class_label":
+                        rule_list = dmge.get_node_validation_rules(node_label=anno_k)
+                    else:
+                        rule_list = dmge.get_node_validation_rules(
```

Comment:
there's also no unit test for `get_node_validation_rules`. Can we also have a ticket for that?